pkgs/development/ocaml-modules/dns/client.nix

```diff
@@ -11,10 +11,25 @@ buildDunePackage {
   pname = "dns-client";
   inherit (dns) src version;
 
-  propagatedBuildInputs = [ cstruct fmt logs dns randomconv domain-name ipaddr
```

Comment:
if you do reformatting prefer a tree-wide one so we can add one commit to ignore in .git-ignore-revs